code/game/objects/structures/watercloset.dm

```diff
@@ -381,7 +383,7 @@
 			else if(H.w_uniform && wash_obj(H.w_uniform))
 				H.update_inv_w_uniform()
 			if(washgloves)
-				SEND_SIGNAL(H, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
```

Comment:
The comsig on H still needs readding. Same thing with the two below this. The comsig higher up only cleans the tile.